src/sentry/api/endpoints/user_notification_settings_details.py

```diff
@@ -29,16 +30,25 @@ def get(self, request: Request, user: User) -> Response:
         """
 
         type_option = validate_type_option(request.GET.get("type"))
+        v2_format = request.GET.get("v2") == "1"
```

Comment:
can we do a `version=2.0` instead. Incase we make more breaking changes in the future, we don't have to keep adding arguments.